app/Relations/HasAlbumThumb.php

```diff
@@ -206,7 +207,7 @@ function (JoinClause $join) use ($userID) {
 			$builder->select(['covered_albums.id AS album_id'])
 				->addSelect(['photo_id' => $bestPhotoIDSelect])
 				->whereIn('covered_albums.id', $albumKeys);
-			if (!AccessControl::is_admin()) {
+			if (!Gate::check('admin')) {
```

Comment:
(continued from above) ..., we can use what we already have here

```suggestion
			if (!$user->isAdmin) {
```

Reply:
Fixed, see from above.